package.json

```diff
@@ -32,27 +32,23 @@
   },
   "homepage": "https://graphql-extended.github.io",
   "dependencies": {
-    "apollo-server-express": "^1.3.6",
-    "apollo-upload-server": "^7.1.0",
+    "apollo-server-express": "^2.15.1",
     "gqlx-js": "^0.2.9",
     "graphql": "^14.4.1",
     "graphql-subscriptions": "^1.1.0",
-    "graphql-tools": "^4.0.5",
     "graphql-type-json": "^0.2.4",
     "subscriptions-transport-ws": "^0.9.16"
   },
   "peerDependencies": {
-    "express": "3.x || 4.x"
+    "express": "^4.17.1"
```

Comment:
4.x would be good here (this is a peer dep so it should be by def. more relaxed; any strict feature that 4.17 has that 4.0 doesn't?

Reply:
an assumption is to reflect the same `express` version like in [apollo-server@2](https://github.com/apollographql/apollo-server/blob/main/packages/apollo-server-express/package.json#L40)

Reply:
Here we see that `express` is actually a `dependency` - so we *should* drop the peer dependency (its already a transitive one now, i.e., no matter what happens `express` is installed).

The only thing left to decide is whether we should remove `express` from `devDependencies` (=> hidden transitive dep) or move it to `dependencies` (=> explicit transitive dep). In any case having the same specifier here is good - thanks for having an eye on this.